tools/pd-ut/ut.go

```diff
@@ -694,7 +701,8 @@ func buildTestBinaryMulti(pkgs []string) error {
 		packages = append(packages, path.Join(modulePath, pkg))
 	}
 
-	p := strconv.Itoa(buildParallel)
+	// We use 2 * parallel for `go build` to make it faster.
```

Comment:
One question, why do we choose  `2 * parallel` to execute `go build`?

Reply:
background:

When building the test binaries, by default it is the number of cores([pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies)):

> -p n
the number of programs, such as build commands or
test binaries, that can be run in parallel.
The default is GOMAXPROCS, normally the number of CPUs available.

When running the 'building binaries stage' of make ut, the CPU usage is not high with the default value. So I think it's a better idea to increase the parallelism.

> why do we choose 2 * parallel to execute go build? not 1.5 or 3.
Choice 2 wasn't much thought or testing, and I think 2 is fine for the time being :) Or do you have any ideas?

Reply:
> Or do you have any ideas?

I have no idea about that. I thought 2 had any special meaning before